common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -230,6 +263,70 @@ public Bundle getRequestBundleForAcquireTokenInteractive(@NonNull final Interact
         );
     }
 
+    /**
+     * Method to construct a request bundle for broker deviceCodeFlowAuthRequest request.
+     *
+     * @param context                         {@link Context}
+     * @param parameters                      input parameters
+     * @param negotiatedBrokerProtocolVersion protocol version returned by broker hello.
+     * @return request Bundle
+     */
+    public Bundle getRequestBundleForDeviceCodeFlowAuthRequest(@NonNull final Context context,
+                                                        @NonNull final DeviceCodeFlowCommandParameters parameters,
+                                                        @Nullable final String negotiatedBrokerProtocolVersion) {
+        final MsalBrokerRequestAdapter msalBrokerRequestAdapter = new MsalBrokerRequestAdapter();
+
+        final BrokerRequest brokerRequest = msalBrokerRequestAdapter.
```

Comment:
>msalBrokerRequestAdapter. [](http://example.com/codeflow?start=44&length=25)

this.getRequestBundleForDeviceCodeFlowTokenRequest() ? why create new instance of MsalBrokerRequestAdapter